cli/setup.py

```diff
@@ -28,7 +28,7 @@ def readme():
     "tabulate>=0.8.2,<=0.8.3",
     "ipaddress>=1.0.22",
     "enum34>=1.1.6",
-    "PyYAML>=5.1.2",
+    "PyYAML==5.2",
```

Comment:
let's pin this only in case of Python 3.4 so that we do not enforce an older version when not strictly required

Reply:
addressed here: https://github.com/aws/aws-parallelcluster/pull/1586